quickwit-cluster/src/cluster.rs

```diff
@@ -109,7 +108,7 @@ pub struct Cluster {
     /// Node ID.
     pub node_id: String,
     /// A socket address that represents itself.
-    pub listen_addr: SocketAddr,
+    pub gossip_listen_addr: SocketAddr,
```

Comment:
Renamed all the socket attributes and parameters to `<service>_<listen/public>_addr` for clarity.

Reply:
That's much better.